src/ApiService/ApiService/model.cs

```diff
@@ -84,4 +85,46 @@ record Node : ITableEntity
 	public Azure.ETag ETag { get; set; }
 	DateTimeOffset? ITableEntity.Timestamp { get; set; }
 
+}
+
+record ProxyForward : ITableEntity 
```

Comment:
you have an old version try updating your repo. this file was moved and the entity mapping is different

Reply:
Shoot. Updating.